### ISSUE TITLE
Add cold startup tests for Reference Browser

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/perf-frontend-issues#25]
### Why/User Benefit/User Problem
RB uses many of the same features as Fenix so its historical upload data could be useful someday: in any case, we should have the capacity to run it.

**Comment by mcomella** [comment]
Triage: blocked on finalizing our cold start tests for Fenix https://github.com/mozilla-mobile/fenix/issues/6507 though RB does have regular uploads and the default ND cold start test running.

**Comment by mcomella** [comment]
(copy-paste) Not a priority: closing.
